pkg/infrastructure/vsphere/vsphere.go

```diff
@@ -0,0 +1,109 @@
+package vsphere
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/infrastructure"
+	"github.com/openshift/installer/pkg/tfvars"
+	"github.com/openshift/installer/pkg/tfvars/vsphere"
+	"github.com/openshift/installer/pkg/types"
+)
+
+// Now the question, do we copy these or just import?
```

Comment:
In the AWS version, I am exporting the symbol in tfvars.go and then importing the tfvars pkg. If you also need that, let's import instead of copy.

Reply:
yeah thats what I ended up doing too